### ISSUE TITLE
tstun: log spam looks buggy

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#559]
When tailscaled starts up (at least on Windows) there's a bunch of:

```
srv: 1.8M/0.0M tstun: warning: you forgot to use SetFilter()! Packet dropped.
srv: 1.8M/0.0M tstun: warning: you forgot to use SetFilter()! Packet dropped.
srv: 1.8M/0.0M tstun: warning: you forgot to use SetFilter()! Packet dropped.
```

... presumably before SetFilter is called for the first time.

But it makes the program look buggy. Maybe we can just reword it to "SetFilter not called yet; packet dropped".

And it probably only needs to log once, not per packet.

**Comment by dshynkev** [comment]
I guess the question is, why log it in the first place? It was supposed to guard against us accidentally forgetting to SetFilter and breaking networking, so logging it only once if it happens all the time during startup is probably not useful. Our tests also tell us if we accidentally break networking, so perhaps I should ensure that and remove the log line altogether?

**Comment by dshynkev** [comment]
In fact, `magicsock_test` does catch that, so I'll just remove the log line.

**Comment by dshynkev** [comment]
Closed via 19867b2b6d8cec4c991130b0a23cd60e09031189.
